index.js

```diff
@@ -42,6 +44,62 @@ var new_client = function(api_key, config) {
     }
   });
 
+  client.initializeStream = function() {
+    this.initialized = false;
+
+    if (this.es) {
+      this.es.close();
+    }
+
+    this.es = new EventSource(this.stream_uri + "/", {headers: {'Authorization': 'api_key ' + this.api_key}});
+    this.features = {};
+
+    var _self = this;
+
+    this.es.addEventListener('put/features', function(e) {
+      if (e && e.data) {
+        _self.features = JSON.parse(e.data);
+        delete _self.disconnected;
+        _self.initialized = true;
+      }
+    });
+
+    this.es.addEventListener('patch/features', function(e) {
+      if (e && e.data) {
+        var patch = JSON.parse(e.data);
```

Comment:
If this failed to parse, that would mean we're sending out broken JSON, which should not happen, right?

Should be log the invalid data in case it does happen and we're not sure why?